config.yaml

```diff
@@ -0,0 +1,34 @@
+# TODO: Rename Dapp into Gnosis Protocol DApp
```

Comment:
Add warning against editing this file

Reply:
Good idea, done

Reply:
feel free to update the texts to your liking when you take over 👌